rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc

```diff
@@ -96,14 +96,14 @@ The project manager and the project owner cannot be deleted. To change them, use
 
 == Reset a User Password
 
-Reset a password if the user forgot it. RPA Manager will send an email containing a new temporary password to the user. The user can then create a new password.
+Reset a password if the user forgot it. RPA Manager sends an email containing a new temporary password to the user. The user can then create a new password.
 
 . Open the *Users* view of the *User Management* module.
 . Click the *Reset Password* icon image:resetpassword-icon.png[key symbol,1.5%,1.5%] in the panel of the user to edit.
 
 == Change Your Password
 
-Change your own password regularly. You will be logged out in the process and have to log in again. The password is used for login to RPA Manager and RPA Builder.
+Change your own password regularly. You are logged out in the process and have to log in again. The password is used for login to RPA Manager and RPA Builder.
```

Comment:
```suggestion
Change your own password regularly. You are logged out of the process and have to log in again. The password is used to log in to RPA Manager and RPA Builder.
```

Reply:
You are logged out in the process [of changing your password] and have to log in again.
This texts will become obsolete soon.